AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -388,12 +393,44 @@ def exposure_control(property: str = 'name') -> str:
         Exposure Control component properties. Requires PostFX Layer component.
           - 'requires' a list of component names as strings required by this component.
             Use editor_entity_utils EditorEntity.add_components(list) to add this list of requirements.\n
+          - 'Enable' Toggle active state of Exposure Control (bool).
+          - 'Enabled Override' Toggle active state of Exposure Control Overrides (bool).
+          - 'ExposureControlType Override' Toggle enable for Exposure Control Type (bool).
+          - 'ManualCompensation Override' Override Factor for Manual Compensation (0.0, 1.0).
```

Comment:
nit: why do this and several other of the properties have the words formatted in this way? can't we just use `- 'Manual Compensation Override'` similar to what we've done with other components to remain consistent

Reply:
This is actually the way it appears in the editor across all components that have overrides.